src/chigame/users/urls.py

```diff
@@ -26,4 +27,5 @@
     path("accept_friend_invitation/<int:pk>", view=accept_friend_invitation, name="accept-friend-invitation"),
     path("decline_friend_invitation/<int:pk>", view=decline_friend_invitation, name="decline-friend-invitation"),
     path("user_history/<int:pk>", views.user_history, name="user-history"),
+    path("search-results", user_search_results, name="user-search-results"),
```

Comment:
This should be view=user_search_results or view.user_search_results so it can properly go into views.py